### PR TITLE
Feature/interpolate name

### LINKED ISSUE #2: As a user… I should should see 7 "days" of story updates for my cat

### LINKED ISSUE #3: As a user...I should be able to make one choice per "day" that affect its morality

### LINKED ISSUE #28: Cat's name from input field not getting interpolated into stored story

### PR DESCRIPTION
**Pull request by qrispi**
Change Summary:
- This merge adds functionality for the user to choose a name for their cat and for the name to be inserted into the story, choices and reactions.
- The name input div is also now conditionally rendered to only appear at the start of the story

Functions/Methods:
- replace: Helper function that runs a reducer over a single story object and inserts the cat name into that story object
- startStory: updates cat name into the store, calls replace for each story object and sends the new object to the story reducer to be updated in store
- addName: reducer action that replaces the entire story array with one that has the user generated cat name interpolated into it

Closes #28, Closes #3, Closes #2

### CODE REVIEW

File: src/components/Story/Story.js [hunk 1]
```diff
@@ -6,64 +6,59 @@ import Result from "../Result/Result";
...
 
 function Story() {
   const dayNum = useSelector((state) => state.day.value);
-  // const checkCatName = useSelector((state) => state.cat.name);
   const story = useSelector((state) => state.storyline);
-  const [catName, setCatName] = useState("");
+  const [ catName, setCatName ] = useState('');
   const dispatch = useDispatch();
 
+  const replace = (obj) => {
```
Comment (jsahim) [inline src/components/Story/Story.js:17]: Such an inventive solution!

**Review by Jesusborjas006 — APPROVED** [review]
Great job figuring out this big issue with naming the cat!

**Review by BriBourassa — APPROVED** [review]
You did such a good job figuring out the interpolation problem- great job!

**Review by jsahim — APPROVED** [review]
Great work on getting this name to display!

**Pull request merged**